### PR TITLE
Fix linter issues

### PR DESCRIPTION
**Pull request by kirillDanshin**

### CODE REVIEW

File: here/enums.go [hunk 1]
```diff
@@ -64,15 +65,16 @@ var ReverseGeocodingMode = &reverseGeocodingList{
 }
 
 // ValueOfReverseGeocodingMode returns value for ReverseGeocodingMode.
-func (reverseGeocodingMode Enum) ValueOfReverseGeocodingMode() string {
+func (routeMode Enum) ValueOfReverseGeocodingMode() string {
```
Comment (abdullahselek) [inline here/enums.go]: I think it's better to update `routeMode` on `func (routeMode Enum) ValueOfRouteMode()` and on `func (routeMode Enum) ValueOfReverseGeocodingMode()` as `mode`.

**Review by abdullahselek — CHANGES_REQUESTED** [review]
Added a single comment please have a look.

File: here/enums.go (hunk 1 above)
Comment (kirillDanshin) [reply here/enums.go]: I think it's done. note: I'm doing it in the github editor, so there might be some typos yet

Comment (abdullahselek) [reply here/enums.go]: Looks good now thanks.

**Pull request merged**